homeassistant/components/climate/services.yaml

```diff
@@ -1,17 +1,17 @@
 # Describes the format for available climate services
 
 set_aux_heat:
-  name: Turn on/off auxiliary heater
-  description: Turn auxiliary heater on/off for climate device.
+  name: Turn on/off emergency heater
```

Comment:
I think we additionally need to define in our developer docs what we mean with aux heat.

If this is a change in semantics it may need an architecture discussion.